t/lib/server.lua

```diff
@@ -426,6 +426,83 @@ function _M._well_known_openid_configuration()
     ngx.say(openid_data)
 end
 
+function _M.google_logging_token()
+    ngx.req.read_body()
+    local data = ngx.decode_args(ngx.req.get_body_data())
+    local jwt = require("resty.jwt")
+    local rsa_public_key = "-----BEGIN PUBLIC KEY-----\nMFwwDQYJKoZIhvcNAQEBBQADSwAwSAJBAKebDxlvQMGyEesAL1r1nIJBkSdqu3Hr\n7noq/0ukiZqVQLSJPMOv0oxQSutvvK3hoibwGakDOza+xRITB7cs2cECAwEAAQ==\n-----END PUBLIC KEY-----"
```

Comment:
Can we declare two variables outside of this method? I noticed that the same pub key is used in other methods.

Reply:
I noticed this problem during the development process. But this public_key is only used by Google Logging related functions? Is it necessary to define it in a public variable in the file?

Reply:
It's not necessary to make them public, but we can put it in the module level (a module-local variable).

Reply:
OK